apps/idam/base/kustomization.yaml

```diff
@@ -8,4 +8,5 @@ resources:
   - ../idam-user-dashboard/idam-user-dashboard.yaml
   - ../idam-hmcts-access/idam-hmcts-access.yaml
   - ../identity/identity.yaml
+  - ../idam-user-disposer/idam-user-disposer.yaml
```

Comment:
There might be some uncertainty regarding the management and monitoring responsibilities for this service.
Is this going to be owned by IDAM or LAU?

Reply:
Hi, This will be managed and monitored by LAU team. Thanks